src/main/resources/i18n/messages.properties

```diff
@@ -861,49 +858,75 @@ project.samples.filter.by-endDate=To
-
 NcbiExportTable.loader-error=A project id is required.
 NcbiExportDetailsView.loader-error=No project id or export id.
 
+# TYPE: NcbiSubmissionRequest (webapp/resources/js/apis/export/ncbi.ts)
+NcbiSubmissionRequest.projectId=BioProject ID
+NcbiSubmissionRequest.projectId.description=BioProject to submit files to.  This project must be created in NCBI prior to this submission.
+NcbiSubmissionRequest.organization=Organization
+NcbiSubmissionRequest.organization.description=Name of organization submitting these samples.
+NcbiSubmissionRequest.namespace=Identifier Namespace
+NcbiSubmissionRequest.namespace.description=Prefix to use for submission identifiers in NCBI.
+NcbiSubmissionRequest.releaseDate=Release Date
+NcbiSubmissionRequest.releaseDate.description=Submission will not be made public until after the chosen release date.
+
+# TYPE: NcbiBioSample
+NcbiBioSample.id=BioSample ID
+NcbiBioSample.accession=NCBI Accession
+NcbiBioSample.bioSample=BioSample
+NcbiBioSample.libraryName=Library Name
+NcbiBioSample.libraryStrategy=Library Strategy
+NcbiBioSample.librarySource=Library Source
+NcbiBioSample.libraryConstructionProtocol=Library Construction Protocol
+NcbiBioSample.instrumentModel=Instrument Model
+NcbiBioSample.librarySelection=Library Selection
+NcbiBioSample.status=Status
+NcbiBioSample.singles=Single Files
+NcbiBioSample.pairs=Paired Files
+
+# TYPE: NcbiSubmission
+NcbiSubmission.id=Submission ID
+NcbiSubmission.project=Project
+NcbiSubmission.state=Status
+NcbiSubmission.submitter=Submitter
+NcbiSubmission.createdDate=Created Date
+NcbiSubmission.organization=Organization
+NcbiSubmission.bioProject=BioProject
+NcbiSubmission.ncbiNamespace=NCBI Namespace
+NcbiSubmission.releaseDate=Release Date
+NcbiSubmission.bioSamples=Samples
+
+# NEW
+CreateNcbiExport.title=NCBI SRA Export Tool
+CreateNcbiExport.details=Submission Details
+CreateNcbiExport.samples=Samples
+CreateNcbiExport.pairs=Paired End Sequencing Files
+CreateNcbiExport.singles=Single End Sequencing Files
+CreateNcbiExport.files.required=Must select a sequencing file.
+CreateNcbiExport.required=This is a required field.
+CreateNcbiExport.submit=Submit
+CreateNcbiExport.default.title=Default Sample Settings
+CreateNcbiExport.default.description=Values selected here will be used as default values for all samples.  Individual samples can can be overwritten.
+
+
+CreateNcbiExport.biosample-id=BioSample ID
+CreateNcbiExport.biosample-id.desc=An identifier block that lists not only the BioSample accession assigned to that record, but also any other external sample identifier, such as that issued by the source database or repository.
+CreateNcbiExport.library_name=Library Name
+#CreateNcbiExport.library_name.description=The submitter's name for this library. <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/" target="_blank">NCBI Documentation</a>.
```

Comment:
Remove these commented out translations if they are not being used

Reply:
Updated in 9e7d5b5dd7c0e7bb9cf25a02042b05630d56d1db